src/context.rs

```diff
@@ -108,16 +110,16 @@ pub struct DatasetInfo {
     pub name: String,
     pub id: String,
     pub gtfs: String,
-    pub gtfs_rt: String,
+    pub gtfs_rt_urls: Vec<String>,
 }
 
 impl DatasetInfo {
-    pub fn new_default(gtfs: &str, gtfs_rt: &str) -> Self {
+    pub fn new_default(gtfs: &str, gtfs_rt_urls: &[String]) -> Self {
```

Comment:
minor: is'n't it a bit better to take `&[&str]` ?

Reply:
`&[&str]` is annoying when you have a `Vec<String>`. You can use https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=108f3ff659bf1438d2bc46a12fd12911 but that's a bit more complicated